### PR TITLE
Added TokenMetadataResponse to types

### PR DESCRIPTION
**Pull request by therealnospoon**
Added TokenMetadataResponse Type

### CODE REVIEW

File: src/types/types.ts [hunk 1]
```diff
@@ -136,6 +136,73 @@ export interface Token {
     tokenStandard: TokenStandard;
 }
 
+export interface TokenMetadataResponse {
```
Comment (ethyi) [inline src/types/types.ts:139]: This doesn't look like it points to our new metadata endpoint with `onChainAccountInfo` and `onChainMetadata`
I also see no usage of this type, do you plan on adding a method for the sdk for it?
https://docs.helius.xyz/solana-apis/token-metadata-api#v0-token-metadata

File: src/types/types.ts [hunk 2]
```diff
@@ -136,6 +136,73 @@ export interface Token {
...
+    mint: string;
```
Comment (ethyi) [inline src/types/types.ts:140]: Should be 'account'

File: src/types/types.ts (hunk 1 above)
Comment (therealnospoon) [reply src/types/types.ts:139]: I realized I used the wrong example to create this type. I have now updated with the correct properties including onChainAccountInfo and legacyMetadata. Thank you.

File: src/types/types.ts (hunk 2 above)
Comment (therealnospoon) [reply src/types/types.ts]: Done!

**Comment by solhosty** [comment]
Closing b/c this is inactive.

**Pull request closed without merging**